docs/rst/developer_manual/installation/sources/linux.rst

```diff
@@ -0,0 +1,236 @@
+
+Asio is a cross-platform C++ library for network and low-level I/O programming, which provides a consistent asynchronous model.
+TinyXML2 is a simple, small and efficient C++ XML parser.
+Install these libraries using the package manager of the appropriate Linux distribution.
+For example, on Ubuntu use the command:
+
+.. code-block:: bash
+
+    sudo apt install libasio-dev libtinyxml2-dev
+
+.. _openssl_sl:
+
+OpenSSL
+^^^^^^^
+
+OpenSSL is a robust toolkit for the TLS and SSL protocols and a general-purpose cryptography library.
+Install OpenSSL_ using the package manager of the appropriate Linux distribution.
+For example, on Ubuntu use the command:
+
+.. code-block:: bash
+
+   sudo apt install libssl-dev
+
+.. _yaml_cpp:
+
+yaml-cpp
+^^^^^^^^
+
+yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec, and is used by *Fast DDS Spy* application to parse the provided configuration files.
+Install yaml-cpp using the package manager of the appropriate Linux distribution.
+For example, on Ubuntu use the command:
+
+.. code-block:: bash
+
+   sudo apt install libyaml-cpp-dev
+
+.. _eprosima_dependencies:
+
+eProsima dependencies
+^^^^^^^^^^^^^^^^^^^^^
+
+If it already exists in the system an installation of *Fast DDS* and *DDS Pipe* libraries, just source this libraries when building |eddsrecord| by running the following commands.
+In other case, just skip this step.
+
+.. code-block:: bash
+
+    source <fastdds-installation-path>/install/setup.bash
+    source <ddspipe-installation-path>/install/setup.bash
+
+
+.. _colcon_installation:
+
+Colcon installation
+===================
+
+#.  Create a :code:`fastdds-spy` directory and download the :code:`.repos` file that will be used to install |eddsrecord| and its dependencies:
+
+    .. code-block:: bash
+
+        mkdir -p ~/fastdds-spy/src
+        cd ~/fastdds-spy
+        wget https://raw.githubusercontent.com/eProsima/Fast-DDS-Spy/main/fastddsspy.repos
+        vcs import src < fastddsspy.repos
+
+    .. note::
+
+        In case there is already a *Fast DDS* installation in the system it is not required to download and build every dependency in the :code:`.repos` file.
+        It is just needed to download and build the |eddsrecord| project having sourced its dependencies.
+        Refer to section :ref:`eprosima_dependencies` in order to check how to source *Fast DDS* library.
+
+#.  Build the packages:
+
+    .. code-block:: bash
+
+        colcon build --packages-up-to-regex fastddsspy
+
+.. note::
+
+    Being based on CMake_, it is possible to pass the CMake configuration options to the :code:`colcon build` command.
+    For more information on the specific syntax, please refer to the `CMake specific arguments <https://colcon.readthedocs.io/en/released/reference/verb/build.html#cmake-specific-arguments>`_ page of the colcon_ manual.
+
+.. TODO check if this is correctly parsed, because in local it does not render correctly
+
+CMake installation
```

Comment:
I would keep the same structure as in other repos. Also, the linked instructions are unindexed.

Reply:
The idea was to degrade cmake option and not show it unless you are really looking for it. This project uses 9 colcon subpackages, it starts to not make much sense to use cmake to compile it.

Reply:
@rsanchez15 ?

Reply:
I would prefer not to. I do not like other repos documentation, it seems indirect and difficult to follow. I would leave CMake only for those experienced users that really want to look for it